rosdep/base.yaml

```diff
@@ -7160,6 +7160,19 @@ osmium:
   fedora: [libosmium-devel]
   nixos: [libosmium]
   ubuntu: [libosmium-dev]
+paho-mqtt-c:
+  alpine: [paho-mqtt-c]
+  debian:
+    bullseye: [libpaho-mqtt1.3]
+  fedora: [paho-c]
+  nixos: [paho-mqtt-c]
+  opensuse: [libpaho-mqtt1]
+  ubuntu:
+    jammy: [libpaho-mqtt1.3]
+paho-mqtt-cpp:
+  nixos: [paho-mqtt-cpp]
+  ubuntu:
+    jammy: [libpaho-mqttpp3-1]
```

Comment:
It is generally recommended to have ubuntu and debian keys, and the rest as needed. Is this package available in debian ?

Reply:
Yes, I know. Unfortunately only the C library is available for Debian Bullseye as included in this PR. The C++ library has been released for Debian Bookworm though, see [libpaho-mqttpp3-1](https://packages.debian.org/bookworm/libpaho-mqttpp3-1). Bookworm is Bullseye's successor, but still in testing though, so I didn't add it to the PR.

Reply:
@adityapande-1995 Gentle reminder to check on my answer :)